### ISSUE TITLE
BOBOT beta testing

### ISSUE THREAD

**Issue by Fruitfella** [Fruitfella/ROBOT-binary#1]
BOBOT adalah robot opsi biner yang sedang saya kembangkan, powered by bot.binary.com
nama BOBOT dimaksudkan agar robot ini nantinya menjadi robot yang berbobot (bermutu), tidak seperti robot abal2 gratisan yang banyak beredar di internet, yang katanya bisa profit ribuan dollar dalam hitungan jam dan hari, tapi mereka nggak bilang kalo robot itu juga bisa ngabisin balance akun kita dalam hitungan detik.

Setelah trial and error, saya berhasil menuangkan ide dan strategi saya kedalam BOBOT. 3 hari alpa testing menunjukan kalau BOBOT dapat berjalan dengan baik tanpa error pada logika robot dalam menjalankan strategi opsi biner.

Setiap test dilakukan selama 5 jam nonstop (stop terjadi ketika stoploss atau takeprofit, tapi test tetap dijalankan kembali sampai waktu minimal 5 jam itu terpenuhi) pada sesi market yang berbeda.

Hasil Alpa Tes : 
Jumlah tes    : 12x
Jumlah trade :  5 tes 150an trade, 6 tes 100an trade dan 1 tes 20an trade
Win/Loss      :  8x stoploss= 5 loss, 3 profit. 4x takeprofit= 4 win.

Settingan dasar BOBOT : 
stoploss = 16
takeprofit= 50
trailling stop =2
initial/win amount =0.35
1 minute Rise/Fall Volatility Indices 100 idx.

================================================================
BOBOT beta testing sudah berjalan mulai 4 Mei 2017 pkl 18.00 wib, tes akan dilakukan selama 2 minggu, 5 jam nonstop tes (setelah 5 jam BOBOT dan laptop saya perlu istirahat sejenak, baru kemudian dilanjutkan lagi tes berikutnya).
Saya akan mencacat detil tes untuk mengetahui dimana kekurangan yang perlu diperbaiki agar BOBOT bisa terus profit dan resiko yang kita tanggung bisa berkurang.

Setting : 
Akun Demo : $6887.35 (ini balannce saat beta tes di mulai)
stoploss = 16
takeprofit= 50
trailling stop =2
initial/win amount =0.5
1 minute Rise/Fall Volatility Indices 100 idx.

P.S. tujuan post ini adalah untuk dokumentasi pribadi.

**Comment by Fruitfella** [comment]
Traksaksi pertama berhenti pada pukul 19.58 wib
Status : Stoploss
No. Runs : 57
Total Stake : 105.3
Total Payout : 107.48
Win : 32
Loss : 25
Profit : 2.18
Balance : 6889.53
/restart

**Comment by Fruitfella** [comment]
best trades : 5x win berturutan
![bes1](https://cloud.githubusercontent.com/assets/26277327/25705987/5df1ebec-3109-11e7-88cc-12fddcfabcab.PNG)
![bes2](https://cloud.githubusercontent.com/assets/26277327/25706000/64b893f4-3109-11e7-8280-2d0d166802ad.PNG)

**Comment by Fruitfella** [comment]
BOBOT berhenti lagi di stoploss dengan loss yg cukup besar, melewati batas stoploss.
kok bisa melewati stoploss? itu karena martingale, sebelum loss terakhir jumlah profit -15an (masih dibawah stoploss) tapi karena martingale bet bertambah tiap loss, dan loss terakhir 9.74, makanya bisa melewati batas stoploss. kerugian ini masih dalam batasan yang dapat kita tanggung tapi saya akan tetap mencoba mencari solusinya.
![ugly2](https://cloud.githubusercontent.com/assets/26277327/25708881/ff8e86b0-3111-11e7-9742-220a9f29cf7c.PNG)
sepertinya keadaan pasar saat ini cukup buruk buat BOBOT.

**Comment by Fruitfella** [comment]
best trades kali ini 6x berturutan
![bes3](https://cloud.githubusercontent.com/assets/26277327/25711107/52fc34f4-3118-11e7-9a6b-4d47a8b26cd2.PNG)
![bes4](https://cloud.githubusercontent.com/assets/26277327/25711116/5a3a7956-3118-11e7-8220-85a13fea1d72.PNG)

**Comment by Fruitfella** [comment]
ada masalah... chartnya berhenti bergerak dan setelah 1 menit masih melum bergerak dan trade juga tidak ditutup. entah salah server, jaringan atau laptop q rusak?
![errorkah](https://cloud.githubusercontent.com/assets/26277327/25714175/57759ecc-3121-11e7-994c-012565a1fa2c.PNG)

**Comment by Fruitfella** [comment]
udah jalan sebentar tapi kemudian macet lagi
![errorkah3](https://cloud.githubusercontent.com/assets/26277327/25714310/bf86fbf0-3121-11e7-95da-25bca6b82215.PNG)

**Comment by Fruitfella** [comment]
kayaknya beta testing harus di stop dlu utk periksa apa masalahnya. kebetulan waktunya jg sudah 5 jam sejak tes pertama td dimulai. cuma harusnya kita nunggu botnya berhenti sendiri (SL/TP).

**Comment by Fruitfella** [comment]
ternyata kena virus, startpage sama search engine di semua browser berubah, tiap kali di ganti balik lg begitu. trus auto download and install sampahware, sampahwarenya gak bs di uninstall ataupun delete.

**Comment by Fruitfella** [comment]
udah buang virus pake malwarebyte.
mau coba lanjutin beta tes. BOBOT mengalami sedikit perubahan. untuk menghindari situasi seperti kemarin dimana BOBOT banyak mengalami loss pada market yg banyak sumbu lilinnya, kondisi untuk melakukan purchase diperketat, jadi jumlah trade yg akan dilakukan BOBOT perjam nya akan berkurang.
ini diharapkan dapat mengurangi jumlah maksimal loss beruntun.
Stop loss dan Trailing profit di hapus, diganti dengan MaxLosingStreak yg merupakan batasan maksimal jumlah loss beruntun yg dapat ditanggung BOBOT. intinya fungsi parameter ini sebenarnya sama dengan gabungan Stop loss dan Trailing profit. jadi lebih simpel.

**Comment by Fruitfella** [comment]
BOBOT ver.0.2 kalah.
meski jumlah win kali ini lebih banyak dari loss tapi sekali 5 loss beruntun, BOBOT kalah dan mengalami kerugian.
1. perlukah menambah batasan MaxLosingStreak? kalau ditambah jadi 6 atau tujuh, mungkin BOBOT bisa membalikkan keadaan, tapi mungkin juga malah semakin banyak kerugiannya.
2. atau mematikan Martingale? dengan jumlah win lebih banyak dari loss, jika BOBOT tidak memakai martingal dan trading dengan jumlah stake yg sama, hasilnya masih bisa profit.
3. atau memperketat lagi syarat purchase.

nb: kok gak bisa upload gambar lagi ya?

**Comment by Fruitfella** [comment]
OK, kompi udah bersih, tapi project bobot untuk sementara di tunda sampai... tak tentu

**Comment by Anoubizz** [comment]
Min ada enggak bot binary yang sekiranya modal dikit tapi profit bnyak?
